lib/catalog/martyrology.ts

```diff
@@ -352,6 +365,13 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       titles: [Titles.Priest],
       dateOfDeath: 1539,
     },
+    antoine_chevrier_priest: {
+      canonizationLevel: CanonizationLevels.Blessed,
+      name: 'Antoine Chevrier',
```

Comment:
Anglicise the first name of Anthody Chevrier ([src](https://leprado.org/wp-content/uploads/2018/12/Mass-of-F.Chevrier.pdf)).

```suggestion
    anthony_chevrier_priest: {
      canonizationLevel: CanonizationLevels.Blessed,
      name: 'Anthony Chevrier',
```